include/vcpkg/commands.build.h

```diff
@@ -162,7 +162,7 @@ namespace vcpkg
         Optional<std::string> public_abi_override;
         std::vector<std::string> passthrough_env_vars;
         std::vector<std::string> passthrough_env_vars_tracked;
-        std::vector<std::string> hash_additional_files;
+        std::vector<Path> hash_additional_files;
```

Comment:
what is the benefit of this?

Reply:
replaces copy of the string into Path with a move